tests/quick.rs

```diff
@@ -253,7 +253,6 @@ where
         let mut it = get_it();
 
         for _ in 0..(counts.len() - 1) {
-            #[allow(clippy::manual_assert)]
```

Comment:
Ah this lint was in "style" category (so warn by default) but was moved in "pedantic" (allow by default), nice.